lib/kernel/doc/guides/eep48_chapter.md

```diff
@@ -145,8 +148,11 @@ information to each entry. This EEP documents the following metadata keys:
 - **`since := binary()`** - a binary representing the version such entry was
   added, such as `<<"1.3.0">>` or `<<"20.0">>`.
 
-- **`edit_url := binary()`** - a binary representing a URL to change the
-  documentation itself.
+- **`source := binary()`** - the location of the source file for this module.
```

Comment:
I think this should go a bit deeper into how this will interact with the `deterministic` option since capturing a local path here will introduce a new source of non-determinism of `.beam` files

Reply:
I also believe, if we're storing location, we should likely call this `path` or similar. To me the name `source` somehow implies this will contain the entire source code as a string.